setup-services.sh

```diff
@@ -32,29 +30,94 @@ sudosed() {
   sed -e "$1" "$2" | $SUDO tee "$3" > /dev/null
 }
 
-# - Install RPM dependencies
-
-# this repo can be removed once python-language-data reaches Factory
-test -f /etc/zypp/repos.d/d_l_python.repo || \
-  $SUDO zypper --non-interactive \
-    addrepo https://download.opensuse.org/repositories/devel:/languages:/python/openSUSE_Tumbleweed/ d_l_python
-$SUDO zypper --non-interactive --gpg-auto-import-keys install gcc gcc-c++ make openssl-devel ruby-devel \
-  python-langtable-data git augeas-devel jemalloc-devel awk suseconnect-ruby-bindings || exit 1
-
-# only install cargo if it is not available (avoid conflicts with rustup)
-which cargo || $SUDO zypper --non-interactive install cargo
-
 # if agama is already running -> stop it
 $SUDO systemctl list-unit-files agama.service &>/dev/null && $SUDO systemctl stop agama.service
 
-# - Install service rubygem dependencies
+# Ruby services
+
+# Packages required for Ruby development (i.e., bundle install).
+$SUDO zypper --non-interactive --gpg-auto-import-keys install \
+gcc \
```

Comment:
I would indent the package names for better readability. It is what we usually do in (e.g., in our Dockerfiles).